0000-evidence-template.md

```diff
@@ -33,6 +34,20 @@
 |   |   |   |   |
 
 
+## Voting record
+*Provide your voting record in relation to required thresholds.* 
+
+|  Ranks | Activity thresholds | Agreement thresholds | Links | Comments |
+|---|---|---|---|---|
+|III|70%   |100%  |[My votes](https://collectives.subsquare.io/user/13aYUFHB3umoPoxBEAHSv451iR3RpsNi3t5yBZjX2trCtTp6/votes) |*This is an example.* I have *[Never/Sometimes/Often/Always]* voted across all referendum tracks AND *[Never/Sometimes/Often/Always]* voted in line with expectations at rank. |
```

Comment:
Note: The manifesto mentions the following:
_"This threshold is a soft requirement: Voters should generally disapprove in cases where the threshold is not met, but if the individual can adequately explain that their voting record is properly in line with a reasonable interpretion of this document, the core tenets and existing precedent, then the disparity may be overlooked."_
 
So, it seems that the assessment of voting records can make or break a request for retention/promotion at rank 3+, however I am not sure it the above example  accurately captures this idea.

Reply:
Evidence should include specific voting activity so that the numbers are clear and voters can make a clear decision.

Reply:
> Evidence should include specific voting activity so that the numbers are clear and voters can make a decision in line with the Manifesto.

Ok, so the voting record of each member needs to be more granular to indicate:
- the activity rate for each relevant track/s (i.e Members, Proficient members, Fellows, etc.) in %, aka _Are they consistently voting on available referenda?_
- the agreement rate for each relevant track/s in %, aka _Are they consistently voting in line with higher ranks?_
- the reasons for any major/minor divergence from the mandated thresholds

Would this breakdown work for making a decision at a glance?

Reply:
we really really need a tool to just show those stats instead of require everyone to review all their voting history and try to figure out the number and then require everyone else to verify the numbers 😓

Reply:
I have updated the guidelines and the form. Please review.